examples/hive/k8s/deploy/hiveserver.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: hiveserver
+spec:
+  type: "LoadBalancer"
+  selector:
+    app: hiveserver
+  ports:
+    - port: 80
+      targetPort: 10002
+      name: web-ui
+    - port: 10001
+      targetPort: 10001
+      name: cliservice
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hiveserver
+  namespace: drlmcdlf
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hiveserver
+  template:
+    metadata:
+      labels:
+        app: hiveserver
+    spec:
+      containers:
+        - name: hiveserver-container
+          image: ${HIVESERVER_IMAGE}
+          imagePullPolicy: "Always"
```

Comment:
Change pull policy, it fails in minikube because it tries pulling the image. **IfNotPresent** should make the trick